index.js

```diff
@@ -56,9 +50,10 @@ function getIndexes(collection) {
  * @param {Collection} collection Mongoose collection.
  * @param {Object} values Hashmap containing data about duplicated values
  * @param {Object} messages Map fields to unique error messages
+ * @param {String} message Default message formatter string
  * @return {Promise.<ValidationError>} Beautified error message
  */
-function beautify(error, collection, values, messages) {
+function beautify(error, collection, values, messages, message) {
```

Comment:
I think we should rename `message` to `defaultMessage` to avoid confusion between the singular `message` and the plural `messages` variable.